man/ctags-optlib.7.rst.in

```diff
@@ -18,4 +18,289 @@ SYNOPSIS
+3. Give a name to the parser
+
+   Use ``--langdef=name`` option. *NAME* is refereed as *<LANG>* in
+   the later steps.
+
+4. Define kinds
+
+   Use ``--kinddef-<LANG>=letter,name,description`` option.
+   Universal-ctags introduces this option.  Exuberant-ctags doesn't
+   have. In Exuberant-ctags, a kind is defined as a side effect of
+   specifying ``--regex-<LANG>=`` option. So user doesn't have a
+   change to recognize how important the definition of kind.
+
+5. Define patterns
+
+   Use ``--regex-<LANG>=/regexp/replacement/[kind-spec/][flags]`` option.
+
+   As *KIND-SPEC*, you can use the letter defined with
+   ``--kinddef-<LANG>=letter,name,description`` option.
+
+EXAMPLE
+------------
+
+This is the definition(pod.ctags) used in ctags for parsing pod
+(https://perldoc.perl.org/perlpod.html) file.
+
+::
+
+   --langdef=pod
+   --map-pod=+.pod
+
+   --kinddef-pod=c,chapter,chapters
+   --kinddef-pod=s,section,sections
+   --kinddef-pod=S,subsection,subsections
+   --kinddef-pod=t,subsubsection,subsubsections
+
+   --regex-pod=/^=head1[ \t]+(.+)/\1/c/
+   --regex-pod=/^=head2[ \t]+(.+)/\1/s/
+   --regex-pod=/^=head3[ \t]+(.+)/\1/S/
+   --regex-pod=/^=head4[ \t]+(.+)/\1/t/
+
+
+OPTION ITEMS
+------------
+
+``--extradef-<LANG>=...``
+	TBW
+
+``--fielddef-<LANG>=...``
+	TBW
+
+``--langdef=name``
+	Defines a new user-defined language, *name*, to be parsed with regular
+	expressions. Once defined, name may be used in other options taking
+	language names.
+
+``--list-regex-flags``
+	Lists the flags that can be used in ``--regex-<LANG>`` option.
+
+``--kinddef-<LANG>=letter,name,description``
+    Define a kind for *<LANG>*.
+	Be not confused this with ``--kinds-<LANG>``.
+
+	*letter* must be an alphabetical character('[0-9a-zA-EG-Z]+')
+	other than "F". "F" has been reserved for representing a file
+	since Exuberant-ctags.
+
+	*name* also must come from alphabetical characters('[0-9a-zA-Z]+').
+	Both *letter*, *name* and their combination must be unique in
+	a *<LANG>*.
+
+	This option is newly introduced in Universal-ctags.  This option
+	reduces the typing defining a regex pattern with
+	``--regex-<LANG>=``, and keeps the consistency of kind
+	definitions in a language.
+
+	The *letter* can be used as an argument for ``--kinds-<LANG>``
+	option to enable or disable the kind. Unless ``K`` field is
+	enabled, the *letter* is used as value in the "kind" extension
+	field in tags output.
+
+	The *name* surrounded by braces can be used as an argument for
+	``--kind-<LANG>`` option. If ``K`` field is enabled, the *name*
+	is used as value in the "kind" extension field in tags output.
+
+	The *description* and *letter* are listed in ``--list-kinds``
+	output. All three elements of kind-spec are listed in
+	``--list-kinds-full`` output. Don't use braces in the
+	*description*. They will be used meta characters in the future.
+
+``--regex-<LANG>=/regexp/replacement/[kind-spec/][flags]``
+	The /regexp/replacement/ pair define a regular expression
```

Comment:
`define` -> `defines`
(maybe)

Reply:
You are correct. I found many "... pair IS" in my corpuses.